code/modules/projectiles/ammunition/magazines.dm

```diff
@@ -256,8 +256,50 @@
 	icon_state = "46x30mmt"
 	ammo_type = /obj/item/ammo_casing/c46x30mm
 	caliber = "4.6x30mm"
-	max_ammo = 20
+	max_ammo = 10
 	multi_sprite_step = 4
+	multiload = 0
+	var/being_loaded = FALSE //A var to check if the mag is already being loaded
+	var/double_loaded = FALSE
```

Comment:
```suggestion
	multiload = 0
	 /// A var to check if the mag is already being loaded
	var/being_loaded = FALSE
	/// There are two reloading processes ongoing? 
	var/double_loaded = FALSE
```
Could use autodocs for some extra clarity about what they do.

Reply:
autodoc?

Reply:
Hover your mouse over a var that has autodoc, it will show the description of it. set by using "/// Comment here" above the var.

Reply:
Done! Thanks